npm-shrinkwrap.json

```diff
@@ -6612,41 +6612,43 @@
       "resolved": "https://registry.npmjs.org/querystringify/-/querystringify-0.0.3.tgz"
     },
     "raml-1-parser": {
-      "version": "1.1.13",
+      "version": "1.1.12",
```

Comment:
Starting from `1.1.13` there are some issues raising. We haven't noticed so far because we already had a previous version of `raml-1-parser` install that satisfied the constraint.